cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1141,417 +1141,441 @@ fpfunc_andacc = 1      ! fixed point function for Anderson acceleration: 1: g(x)
-            write(nu_diag,1012) ' basalstress      = ', basalstress,trim(tmpstr2)
+            write(nu_diag,1010) ' basalstress      = ', basalstress,trim(tmpstr2)
             if (basalstress) then
-               write(nu_diag,1007) ' k1               = ', k1, ' free parameter for landfast ice'
-               write(nu_diag,1007) ' k2               = ', k2, ' free parameter for landfast ice'
-               write(nu_diag,1007) ' alphab           = ', alphab, ' factor for landfast ice'
-               write(nu_diag,1007) ' threshold_hw     = ', threshold_hw, ' max water depth for grounding ice'
+               write(nu_diag,1002) ' k1               = ', k1, ' : free parameter for landfast ice'
+               write(nu_diag,1002) ' k2               = ', k2, ' : free parameter for landfast ice'
+               write(nu_diag,1002) ' alphab           = ', alphab, ' : factor for landfast ice'
+               write(nu_diag,1002) ' threshold_hw     = ', threshold_hw, ' : max water depth for grounding ice'
+            endif
+            write(nu_diag,1002) ' Ktens            = ', Ktens, ' : tensile strength factor'
+
+            if (kdyn == 3) then
+               write(nu_diag,1020) ' maxits_nonlin    = ', maxits_nonlin,' : max nb of iteration for nonlinear solver'
+               write(nu_diag,1030) ' precond          = ', trim(precond),' : preconditioner for fgmres'
+               write(nu_diag,1020) ' dim_fgmres       = ', dim_fgmres,' : size of fgmres Krylov subspace'
+               write(nu_diag,1020) ' dim_pgmres       = ', dim_pgmres,' : size of pgmres Krylov subspace'
+               write(nu_diag,1020) ' maxits_fgmres    = ', maxits_fgmres,' : max nb of iteration for fgmres'
+               write(nu_diag,1020) ' maxits_pgmres    = ', maxits_pgmres,' : max nb of iteration for pgmres'
+               write(nu_diag,1010) ' monitor_nonlin   = ', monitor_nonlin,' : print nonlinear residual norm'
+               write(nu_diag,1010) ' monitor_fgmres   = ', monitor_fgmres,' : print fgmres residual norm'
+               write(nu_diag,1010) ' monitor_pgmres   = ', monitor_pgmres,' : print pgmres residual norm'
+               write(nu_diag,1030) ' ortho_type       = ', trim(ortho_type),' : type of orthogonalization for FGMRES'
+               write(nu_diag,1009) ' reltol_nonlin    = ', reltol_nonlin,' : nonlinear stopping criterion'
+               write(nu_diag,1009) ' reltol_fgmres    = ', reltol_fgmres,' : fgmres stopping criterion'
+               write(nu_diag,1009) ' reltol_pgmres    = ', reltol_pgmres,' : pgmres stopping criterion'
+               write(nu_diag,1030) ' algo_nonlin      = ', trim(algo_nonlin),' : nonlinear algorithm'
+               write(nu_diag,1010) ' use_mean_vrel    = ', use_mean_vrel,' : use mean of previous 2 iterates to compute vrel'
+               if (algo_nonlin == 'anderson') then
+                  write(nu_diag,1020) ' fpfunc_andacc    = ', fpfunc_andacc,' : fixed point function for Anderson acceleration'
```

Comment:
`fpfunc_andacc` has only one allowed value (the code will abort if it's not "1"), so I would have left it in the "Other ice_in namelist parameters" section...

Reply:
I think it's fine here...